test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests1/Tests/ActionsTests.cs

```diff
@@ -2022,7 +2022,7 @@ public void ActionResolveOrderTests()
                 {
                     RequestUri = "/Foo(1)/Bar",
                     Payload = "{ \"ID\" : 100 }",
-                    ExpectedResponse = "{\"@odata.context\":\"http://host/$metadata#Foo/$entity\",\"ID\":100}",
+                    ExpectedResponse = "{\"@odata.context\":\"http://host/$metadata#Foo(1)/Bar/$entity\",\"ID\":100}",

```

Comment:
The correct context url should be: {type-name} for an entity with optional navigation target.